libs/utils/src/tabulation.ts

```diff
@@ -825,7 +825,8 @@ export function mergeManualWriteInTallies(
   }
 
   return {
-    ...manualResults,
+    // eslint-disable-next-line vx/gts-spread-like-types
+    ...anyResults,
```

Comment:
This seems like an unfair callout by the linter, I'm happy to disable this. `anyResults` extends an `Object` type so it should be spreadable.